nixos/modules/services/networking/akkoma.nix

```diff
@@ -0,0 +1,522 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.akkoma;
+  format = pkgs.formats.elixirConf { };
```

Comment:
Oh, nixpkgs offers a way of generating an elixir config?

Reply:
It does. It took me a while to understand how it works.